### PR TITLE
Milestone/wireframe

### PR DESCRIPTION
**Pull request by shepshook**
Hello!
I'd like to get your approval on the theme I've chosen. Basic features of the app are described in the main readme file (in short, it is a web implementation of [Boosted](https://www.boostedproductivity.com/) app). Also there is some Figma mockups and simple HTML pages in the PR. I'd like to know your opinion if any features should be added.

### CODE REVIEW

**Review by gmltA — CHANGES_REQUESTED** [review]
The topic would do. Keep in mind that you are already behind the first deadline.

Milestone 2 [acceptance criteria](https://github.com/gmltA/FrontEnd_tasks#acceptance-criteria) are not met: I don't see HTML on your mockups.

**Comment by shepshook** [comment]
@gmltA Sorry, but we have no info about any deadlines.

So html tags should be provided directly on mockup? I have separate html files with some main pages, won't it work here?

**Comment by gmltA** [comment]
> @gmltA Sorry, but we have no info about any deadlines.

9th of February was the date I asked for the topics of project. As of 10th of March I still had no information regarding your plans. Consider this deadline a 'surprise mechanics'.

> So html tags should be provided directly on mockup? I have separate html files with some main pages, won't it work here?

You have a description of expected deliverables and acceptance criteria for the first milestone. Images without markup won't do the thing here.

**Comment by shepshook** [comment]
@gmltA, I've updated the Figma mockups with a section with HTML tags description. Could you take a look please?
https://www.figma.com/file/yeygRKutgI5ubootVQI9hF/Intensio-Wireframe

**Comment by gmltA** [comment]
> @gmltA, I've updated the Figma mockups with a section with HTML tags description. Could you take a look please?
> https://www.figma.com/file/yeygRKutgI5ubootVQI9hF/Intensio-Wireframe

Please, re-request a review when you want me to look at your work next time.

File: details.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <nav>
+        <img src="res/logo.svg" alt="">
+        <h3 class="logo">intensio</h3>
+    </nav>
+    <main class="project-details">
+        <h1>Project name</h1>
+        <section>
+            <h2>Today</h2>
+            <div>
+                <p>20:00 - 20:15</p>
```
Comment (gmltA) [inline details.html]: Use `<time>` for such elements.

**Review by gmltA — COMMENTED** [review]
It's <kbd>8</kbd> for now. Tell me if you are willing to improve it.

Keep in mind, that I'll need to see all your markup on the next milestone. You might want to remove it from this PR before merging it.

File: details.html (hunk 1 above)
Comment (shepshook) [reply details.html]: Thanks for the review! I'm good with an 8. 

I removed the html files from here so now I'm merging this PR into `master`.

**Pull request merged**